src/lib/constants.js

```diff
@@ -14,6 +14,9 @@ export const DEFAULT_DEADLINE = 30;
 export const MIN_DEADLINE = 1;
 export const MAX_DEADLINE = 1000;
 
+export const WRAPPED_ETHEREUM_CONTRACT_ADDRESS =
+  'ct_ryTY1mxqjCjq1yBn9i6HDaCSdA6thXUFZTA84EMzbWd1SLKdh';
+
```

Comment:
Only the mainnet address is listed here. DEX supports both testnet and mainnet.

Or 

```suggestion
export const WRAPPED_ETHEREUM_MAINNET_CONTRACT_ADDRESS =
  'ct_ryTY1mxqjCjq1yBn9i6HDaCSdA6thXUFZTA84EMzbWd1SLKdh';

```